docs/markdown/01-intro/05-JSX.md

```diff
@@ -0,0 +1,291 @@
+<!-- .slide: class="transition-bg-blue-3 right"-->
+
+# JSX
+
+##==##
+
+# What is JSX ? what we've seen so far.
+
+- JSX is a syntax extension to JavaScript<br/><br/>
+- Javascript was in charge of HTML<br/><br/>
+- HTML, CSS , and Javascript in the same place<br/><br/>
+
+```jsx
+import './index.css';
+
+const element = <h1>Hello, world!</h1>;
+```
+
+<!-- .element: class="big-code"-->
+
+##==##
+
+<!-- .slide: class="two-column"-->
+
+# The Rules of JSX
+
+- JSX is often compared to the _HTML_ part of React, so it's only natural that JSX shares the same rules as HTML. But since JSX is a syntax extension of JavaScript, there are some notable differences, as you will see below.
```

Comment:
suggestion: this is really important! That's really tru we don't write jsx code as html. So for me it's not just a bullet point in a list it's a message important. Maybe we can remove the bullet point and add the important class like that

```markdown
JSX is often compared to the _HTML_ part of React, so it's only natural that JSX shares the same rules as HTML. But since JSX is a syntax extension of JavaScript, there are some notable differences, as you will see below.
<!-- .element: class="imporant"-->
```

What do you think ?

Reply:
Yep, sounds right to me